### DISCUSSION TITLE
Genstar Document tation

### DISCUSSION THREAD

**Discussion by ngohienduong** [COMOKIT/COMOKIT-Model#47] [Q&A]
@chapuisk Hi Kevin, is there anywhere I can find the documentation for Genstar population generation?

**Comment by chapuisk** [comment]
Hi @ngohienduong, the main project that has been used to generate synth pop for COMOKIT in Vietnam is [here](https://github.com/ANRGenstar/template/blob/master/src/main/java/vietnam/gospl/CoVid19_mainSP.java). It will requires you to identify yourself to [IPUMS services](https://international.ipums.org/international/) in order to download the sample for VN. Aggregated data comes from GSO and has been scattered to fit the 3 case studies of the beginning.
Synth pop of the case studies in Fr has been manage differently, mainly because previous algorithm was very slow (few minutes to generate a population of 10k agents) using only aggregated data, see e.g. the repo of COMOKIT-Azur. The process is based on Gen* for the generation of synthetic individual, that has been fed into a gaml algorithm to construct household based on aggregated statistics (e.g. average number of person in household based on age/gender of the head) and rules of thumbs (e.g. age gap between parent and children).
